content/bn/service-proxy.md

```diff
@@ -0,0 +1,22 @@
+---
+title: সার্ভিস প্রক্সি (Service Proxy)
+status: Completed
+category: প্রযুক্তি
+---
+
+## এটা কি?
+
+একটি সার্ভিস প্রক্সি  একটি প্রদত্ত পরিষেবাতে ([service](/service/)) বা সেখান থেকে ট্র্যাফিককে বাধা দেয়, এতে কিছু যুক্তি প্রয়োগ করে, তারপর সেই ট্র্যাফিকটিকে অন্য পরিষেবাতে অগ্রস্থ করে। এটি মূলত একটি "গো-বিটুইন (go-between)" হিসাবে কাজ করে যা নেটওয়ার্ক ট্র্যাফিক সম্পর্কে তথ্য সংগ্রহ করে এবং/অথবা এটিতে নিয়ম প্রয়োগ করে।
+
+## এটা যেসব সমস্যাতে দৃষ্টিপাত করে 
+
+পরিষেবা যোগাযোগে পরিষেবার ট্র্যাক রাখতে (উপনাম নেটওয়ার্ক ট্র্যাফিক) এবং সম্ভাব্য রূপান্তর বা পুনঃনির্দেশ করতে, আমাদের ডেটা সংগ্রহ করতে হবে।
+ঐতিহ্যগতভাবে, ডেটা সংগ্রহ এবং নেটওয়ার্ক ট্রাফিক ব্যবস্থাপনা সক্ষম করে এমন কোড প্রতিটি অ্যাপ্লিকেশনের মধ্যে অনুবিদ্ধ(embedded)করা হয়েছিল।
+
+## এটা কিভাবে সাহায্য করে
+
+একটি সার্ভিস প্রক্সি আমাদের এই কার্যকারিতা "বহিরাগত" করার অনুমতি দেয়। এটিকে আর অ্যাপের মধ্যে থাকার দরকার নেই। পরিবর্তে, এটি এখন প্ল্যাটফর্ম স্তরে অনুবিদ্ধ করা হয়েছে (যেখানে আপনার অ্যাপগুলি চলে)।
+
+পরিষেবাগুলির মধ্যে দারোয়ান হিসাবে কাজ করা, প্রক্সিগুলি কী ধরনের যোগাযোগ ঘটছে তার অন্তর্দৃষ্টি প্রদান করে৷ তাদের অন্তর্দৃষ্টির উপর ভিত্তি করে, তারা নির্ধারণ করে যে কোথায় একটি নির্দিষ্ট অনুরোধ পাঠাতে হবে বা এমনকি সম্পূর্ণরূপে অস্বীকার করতে হবে।
```

Comment:
It would be great if you make it a simple sentence "পরিষেবাগুলির মধ্যে দারোয়ান হিসাবে কাজ করা, প্রক্সিগুলি কী ধরনের যোগাযোগ ঘটছে তার অন্তর্দৃষ্টি প্রদান করে"